substrate/frame/examples/basic/src/lib.rs

```diff
@@ -131,43 +147,38 @@ impl<T: pallet_balances::Config> PaysFee<(&BalanceOf<T>,)> for WeightForSetDummy
 	}
 }
 
-// Definition of the pallet logic, to be aggregated at runtime definition through
-// `construct_runtime`.
+// Definition of the pallet logic, to be aggregated in a chain's runtime definition.
 #[frame_support::pallet]
 pub mod pallet {
 	// Import various types used to declare pallet in scope.
 	use super::*;
 	use frame_support::pallet_prelude::*;
 	use frame_system::pallet_prelude::*;
 
-	/// Our pallet's configuration trait. All our types and constants go in here. If the
-	/// pallet is dependent on specific other pallets, then their configuration traits
-	/// should be added to our implied traits list.
+	/// Our pallet's configuration trait.
 	///
-	/// `frame_system::Config` should always be included.
+	/// Because all FRAME pallets depend on some core utility types from the System pallet,
+	/// [`frame_system::Config`] should always be included. This pallet example uses the `Balances`
+	/// type from `pallet_balances` which we make available by bounding this trait with
+	/// `pallet_balances::Config`.
```

Comment:
Someday once we have the ref doc for loose vs. tight coupling, we should link to that page.